app/server/schemas.py

```diff
@@ -407,7 +407,7 @@ class TransferUsageSchema(Schema):
 
 transfer_accounts_schema = TransferAccountSchema(
     many=True,
-    only=('balance', 'created', 'id', 'users', 'token', 'primary_user_id', 'blockchain_address')
+    only=('balance', 'created', 'id', 'users', 'token', 'primary_user_id', 'blockchain_address', 'is_approved')
```

Comment:
`is_approved` wasn't being returned in transfer account schema but was being accessed on that page...